### ISSUE TITLE
Incompatible with latest aiohttp.

### ISSUE THREAD

**Issue by ybbbbt** [wookayin/gpustat-web#19]
Hi,

I just find latest `aiohttp==3.8.2` is not compatible with current version.

A temporary workaround is to specify an older version of `aiohttp` in `setup.py`, e.g., changing from `'aiohttp>=3.0.0,<4.0',` to 'aiohttp==3.7.4.post0'.

**Comment by wookayin** [comment]
I can also confirm that `aiohttp<3.8` is required to work. With `aiohttp>=3.8` (2021-10-31), nothing prints. I will look into this.

**Comment by wookayin** [comment]
It's very weird and I don't like aiohttp's breaking change.

As in #7, `app.loop` was deprecated since aiohttp 3.5.0 (was removed from 4.0.0, which is still alpha) so I had to change from `app.loop` to `loop` (fd501df3db98e6f433f062cdf3b749a68ba3c6b7) so that it could work with aiohttp 4.0.0+. However, this breaks in aiohttp 3.8.0 (through 3.7 was fine..) 😠 --- these two are different event loop objects.

Changing it back to `app.loop` would work for all `aiohttp < 4.0`, but aiohttp 4.0.0 will be another trouble. Or we could just require `aiohttp >= 4.0`, but I would avoid using alpha versions yet.
